### PR TITLE
Resolves #31 - Petition for Expungement (Special Certificate from BCI)

### LINKED ISSUE #31: Form - Petition for Expungement (Special Certificate from BCI)
https://www.utcourts.gov/howto/expunge/docs/01_Petition_to_Expunge_Records_Criminal-special_certificate.pdf

### PR DESCRIPTION
**Pull request by kthompson55**
Adds the expungement form for a special certificate from BCI.
There's a lot of conditional logic in this. A quick desk check with @tuckersamuelsen looked good in regards to the proper checkmark/fields showing up based on the user inputs on the input form.

### CODE REVIEW

File: frontend/app/forms/special-bci/special-bci.web.component.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+          />
+        )}
+        <TextInput dataKey="case.caseNumber" label={__("case number")} />
+        <TextInput dataKey="case.judgeName" label={__("judge full name")} />
+      </Section>
+
+      <Section name="3. Records of Crime">
+        <Radio
+          dataKey="case.hasConviction"
+          label={__("have conviction")}
```
Comment (seanbw89) [inline frontend/app/forms/special-bci/special-bci.web.component.js]: This label could be more explicit something along the lines of `Record of conviction`

File: frontend/app/forms/special-bci/special-bci.web.component.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+            dataKey="case.courtAddress"
+            label="Justice Courts"
+            groupOptions={JusticeCourtList}
+          />
+        )}
+        <TextInput dataKey="case.caseNumber" label={__("case number")} />
+        <TextInput dataKey="case.judgeName" label={__("judge full name")} />
+      </Section>
+
+      <Section name="3. Records of Crime">
```
Comment (seanbw89) [inline frontend/app/forms/special-bci/special-bci.web.component.js]: In section 3 include explanation that is included in the PDF.
> Choose (1) and complete that section if you were __not__ convicted of the crime you are asking to expunge.
Choose (2) and complete that section if you __were__ convicted of the crime(s) you are asking to expunge. 
Also complete Paragraph (3) also.

File: frontend/app/forms/bank/form-bank.component.js [hunk 3]
```diff
@@ -211,7 +213,6 @@ export default class FormBank extends React.Component {
           path={`${this.props.match.url}/reply-to-statement`}
           render={props => (
             <GovernmentForm
-              name="Reply to statement or response"
```
Comment (kthompson55) [inline frontend/app/forms/bank/form-bank.component.js]: Accidentally deleted this line somehow. I will need to put this back in.

File: frontend/app/forms/special-bci/special-bci.pdf.component.js [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+const farLeft = "11.28%";
+const petitionerCheckboxLeft = "20.75%";
+const attorneyTop = "33.00%";
+const courtTypeTop = "36.4%";
+const courtDistrictTop = "39.30%";
+const caseInformationLeft = "54.00%";
+
+export default function SpecialBci_Pdf({ data, renderData }) {
+  const firstPageUrl =
```
[the remark this answers is not in the document]
Comment (kthompson55) [reply frontend/app/forms/special-bci/special-bci.pdf.component.js]: I think the way I have it now makes it easier when scrolling through the fairly large pdf components to see which page the RenderPage component is displaying. `firstPageUrl` is easier to read imo than finding the `-1.png` that is appended by the pdf-to-png converter we use.

File: frontend/app/forms/special-bci/special-bci.web.component.js (hunk 2 above)
Comment (joeldenning) [reply frontend/app/forms/special-bci/special-bci.web.component.js]: agreed on this 👍

File: frontend/app/forms/special-bci/special-bci.pdf.component.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (joeldenning) [reply frontend/app/forms/special-bci/special-bci.pdf.component.js]: yeah I think either way is fine. I personally would inline the string, but it doesn't matter much. Resolving this comment.

File: frontend/app/forms/bank/form-bank.component.js [hunk 5]
```diff
@@ -267,6 +269,66 @@ export default class FormBank extends React.Component {
             />
           )}
         />
+        <Route
+          path={`${this.props.match.url}/proof-of-completed-service`}
```
Comment (joeldenning) [inline frontend/app/forms/bank/form-bank.component.js]: We were just missing routes for these other forms? Why are these being added here?

Comment (kthompson55) [reply frontend/app/forms/bank/form-bank.component.js]: Result of a bad conflict resolution. These were duplicates.

**Pull request merged**